src/blocks/info-box/edit.js

```diff
@@ -34,7 +34,8 @@ const UAGBInfoBox = ( props ) => {
 		},
 		name,
 		clientId,
-		deviceType
+		deviceType,
+		context
```

Comment:
Always use trailing comma,

```suggestion
		context,
```